### ISSUE TITLE
[Feature Request / Automation] propose new condition 

### ISSUE THREAD

**Issue by GreenMystic** [aistra0528/Hail#257]
problem:
I don't need gms and play store enabled all the time 
i only need it when opening apps that's really need it (ex: GAPPS, games with cloud save)

Please Consider follow solution:
1. from Hail homepage add option to activate specific app when you are about to launch app
2. from Hail homepage add option to instant launch

note:
1. intant launch mean hail will immediate unfreeze and launch app but hail will freeze the app again if fully closed
(Force Stopped, swiped out from recent)

Thanks for reading and considering developer 🙏

@aistra0528

**Comment by sameerasw** [comment]
An integration for the Tasker app would be so beneficial

**Comment by sameerasw** [comment]
> > An integration for the Tasker app would be so beneficial
> 
> macrodroid as well i paid the app and it worth a shot

yes, my idea is to automate this process so like the iOS focus profiles, I can make my device freeze unwanted apps depending I'm at work, uni, home or such and especially at night....

**Comment by aistra0528** [comment]
Activate GMS tag when you are about to launch app:
`am start -a com.aistra.hail.action.LAUNCH -e package package.need.gms -e tag GMS`
